functions.php

```diff
@@ -128,14 +128,58 @@ function createOptionElement(array $array): string
 
 function addCollectionItem(PDO $pdo): string
 {
-    //1. Get location_id
+    // 1. Get location_id
     $query = $pdo->prepare('SELECT `location_id` FROM `locations` WHERE `country_name`= :country_name AND `region`= :region;');
     $query->bindParam(':country_name',$_POST['country']);
     $query->bindParam(':region',$_POST['region']);
 
-    //2. Execute query
+    // 2. Execute query
     $query->execute();
-    $location_id = $query->fetchAll();
+    (int) $location_id = $query->fetchAll();
+
+    // 3. Insert Statement into `routes`
+    $query = $pdo->prepare(
+'INSERT INTO `routes` (`name`, `location_id`, `distance`, `discipline`, `effort_level`, `adrenaline_rating`, `short_description`)'.
```

Comment:
Indent SQL statement for easier readability. Check line 169 for better notion of what to aim for!